### ISSUE TITLE
Add Documentation Links

### ISSUE THREAD

**Issue by jonshern** [guregu/dynamo#115]
Wanted to get this package added to awesome go but it is missing a couple of things in the Readme

godoc link to the repo
coverage service link to the repo 
goreportcard link to the repo

**Comment by auyer** [comment]
I agree this package deserves to be in Awesome, and I can help you with that @guregu 

The parts I can not are setting up the integration between your CI and Coverage tool. I see you have a CircleCI file, but its either private or not working. 
If you'd like, I can add the goreportcard, CI and Coverage badges, and the CircleCI configuration. You would just need to connect the accounts ! 
This repo of mine have such things and you can give it a look : [auyer/steganography](https://github.com/auyer/steganography).
